package.json

```diff
@@ -20,18 +20,21 @@
   },
   "homepage": "https://github.com/jpitucha/houston#readme",
   "dependencies": {
-    "@types/lodash": "^4.14.167",
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "joi": "^17.3.0",
```

Comment:
Remove joi and @types/joi from dependencies, because you are using superstruct instead.